book/src/frameworks.md

```diff
@@ -23,6 +23,15 @@ The crates [`thiserror`] and [`displaydoc`] can be useful to implement this erro
 [`thiserror`]: <https://crates.io/crates/thiserror>
 [`displaydoc`]: <https://crates.io/crates/displaydoc>
 
+> Alternatively, you can use [`#[derive(askama_web::WebTemplate)]`](https://crates.io/crates/askama_web/)
```

Comment:
Just realized: why are you putting this text in a quote?

Reply:
I wanted to visually separate the block a bit. I replaced the blockquote it with a headline, because it just does that, too. :)